wordpress-mobile/gutenberg-mobile
# Test Aztec 1.17.1

Test Aztec 1.17.1

Run 'pod install' inside the 'ios' folder.

If needed delete the `ios/Pods` folder if 'pod install' fails.

To test:
 - Open demo app
 - Edit a paragraph block
 - Add a link to the end of content
 - Check that the underline is drawn on all content.

PR submission checklist:
